python/search_widget/search_widget.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, parent=None):
         # set up the UI
         self._ui = Ui_SearchWidget()
         self._ui.setupUi(self)
-        self.set_placeholder_text("Search")
+        self.set_placeholder_text("Search...")
```

Comment:
Type here to search... might be too long for certain UIs. I'd rather stick with "Search..." for now.